website/blog/2020-06-01-build2020recap.md

```diff
@@ -0,0 +1,83 @@
+is_blog: true
+published_date: June 01, 2020
+---
+
+A little over a week ago, Microsoft wrapped up the annual [Build developer conference](https://mybuild.microsoft.com/). React Native was a big theme at Build 2020 this year, with over 6 sessions dedicated to React Native and many tie-ins from other talks.
+
+The biggest announcement from the team is that React Native for macOS is ready for general consumption! Although we announced this on [our blog](2020-05-19-rn4mupdadates.md) a few weeks ago, the team spent more time deep-diving into how folks can get started building apps for Windows and macOS.
+
+We also announced several 1st and 3rd party applications that are taking advantage of React Native. For example, we showed how the Xbox (Beta) app for PC uses React Native for Windows to create great native experiences for gamers. Additionally, we announced that Facebook was using React Native for Windows and macOS to build Messenger Desktop. 
+
+[![Gif of Xbox App](assets/build2020_xbox.gif)](https://www.microsoft.com/store/productId/9MV0B5HZVK9Z)
+
+## Key Takeaways
+
+The team held 2 focus groups focused on cross-platform native app development, where the main goal was to understand motivotors, invesments, and challenges that people developing cross-platform applications faced. Below are some key takeaways from these focus groups and Q&A sessions:
+
+- Cross-platform developers are looking to share as much code as possible between mobile and desktop apps; however, developers do differentiate between mobile (iOS, Android) and Desktop (Windows, macOS, Linux). 
+
+- React Native continues to gain momentum and is seen as a viable stack due to familiaritiy with React and industry investment; this interest has become so prevalent that **enterprise companies are requesting solutions built with React Native by name.**
+
+- **Linux was a common target platform** for developers that are looking to target desktop. 
+
+- Compared to Electron, native-based platforms like React Native were perceived favorably mainly due to performance. - **"I avoid Electron and do NOT recommend it!"**
+
+- Developers and designers prefer native apps; platforms like Flutter are gaining in populatority, but not being native is seen as a negative. - **"Native look and feel is very important. Flutter loses out here. Has to look like a Native app. Uncanny valley with users if not."**
+
+## Sessions
+
+### Skilling Session - "Build cross platform apps that target Windows, Mac, and more!"
+
+React Native isn't just for building mobile apps! Come learn how you can use React Native to build truly native applications that target the entire family of Windows devices, Android, iOS, and Mac!
+
+View the session on [YouTube](https://www.youtube.com/watch?v=QMFbrHZnvvw).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/QMFbrHZnvvw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Does youtube's iframe register on the localhost site you run to test this? I haven't actually tried this type of embedding before

Reply:
Yup, this seems to work ok!

![image](https://user-images.githubusercontent.com/25991996/83459398-2891c180-a419-11ea-97d4-e7df7d41ef11.png)